src/GHC/RTS/Events/Binary.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE ViewPatterns #-}
+{-# LANGUAGE TypeApplications #-}
```

Comment:
This is causing CI for 7.10.3 to fail. Maybe we can just drop support for GHC 7.10.3.